### ISSUE TITLE
website-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.js-playgroundShareEl) Tags: Accessibility, WCAG 1.4.3, color-contrast

### ISSUE THREAD

**Issue by A11Ybot-OP** [golang/go#53564]
website https://go.dev/
Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: The Go Programming Language - https://go.dev/

Element path: .js-playgroundShareEl

Snippet: `<button class="Button js-playgroundShareEl" title="Share in Go Playground">Share</button>`

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.48 (foreground color: #007d9c, background color: #f8f8f8, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

Environment: Chrome version 102.0.0.0

====

There are 28 other WCAG 1.4.3 issues on the website.

**Comment by dr2chase** [comment]
This is waiting on an update to the L&F for playground.
